src/Sentry/Platforms/Android/SentrySdk.cs

```diff
@@ -179,7 +165,7 @@ private static void InitSentryAndroidSdk(SentryOptions options)
                 o.AddIgnoredExceptionForType(JavaClass.ForName("android.runtime.JavaProxyThrowable"));
             }));
 
-        // Set options for the managed SDK that depend on the Android SDK
+        // Set options for the managed SDK that depend on the Android SDK. (The user will not be able to modify these.)
```

Comment:
> The user will not be able to modify these

does this mean "there is no API what the user can use to modify these" or "we ignore/overwrite the options the user sets"?

Reply:
It means, we set these options ourselves during init, ignoring whatever the user provided.  These should be limited to what is needed to make the native and managed SDKs work together.